### PR TITLE
Breaking Down Docker Containers vs. Virtual Machines (VMs)

### LINKED ISSUE #2164: Breaking Down Docker Containers vs. Virtual Machines (VMs)
### Proposed title of article
Breaking Down Docker Containers vs. Virtual Machines (VMs)

### Introduction paragraph (2-3 paragraphs):
Virtual Machines (VMs) and containers can help get the most out of available computer hardware and software resources. Containers are the new kids on the block, but VMs have been, and continue to be, tremendously popular in data centers of all sizes.

Virtual machines and containers are used to create isolated virtual environments for developing and testing applications or software. The question is how they differ.

This article examines the two concepts to help understand the difference between a container and a VM. It defines containers and virtual machines separately and side-by-side to see what makes one distinct from the other.

### Key takeaways: 

1. How VMs work
2. How containers work
3. Containers pros and cons
4. VMs pros and cons  
5. Which one to use: Containers vs. Virtual Machines

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.
N/A
### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by nellyat**
this fixes #2164

### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

### CODE REVIEW

File: articles/breaking-down-docker-containers-vs-virtual-machines/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+Containers(/engineering-education/history-of-container-technology/) and virtual machines (VMs) help maximize the value you get from the available software resources and computer hardware. Virtual machines are historically popular for their ability to allow users to run multiple operating systems simultaneously and share the same hardware resources. VMs are run by a hypervisor that allows a single host computer to support more than one guest virtual machine by sharing processing, memory, and other resources.
```
Comment (WanjaMIKE) [inline articles/breaking-down-docker-containers-vs-virtual-machines/index.md:1]: Links is ref like this ```[containers](/engineering-education/history-of-container-technology/)```

**Review by WanjaMIKE — CHANGES_REQUESTED** [review]
Please check the above comment. Thanks

**Comment by WanjaMIKE** [comment]
@nellyat please add author information. Thanks

**Comment by WanjaMIKE** [comment]
@nellyat did you archive this repo? I cannot make a commit

**Comment by WanjaMIKE** [comment]
Ensure that the repository or branch is public. Currently, I cannot make any commits. Thanks

**Comment by WanjaMIKE** [comment]
Kindly, attend to the revision as soon as possible or the article will be closed. Thanks

**Comment by hectorkambow** [comment]
Thank you v much for your patience as we worked through our Repo re-structuring process. 

The purpose behind this restructure - was to better allow us to handle more volume and allow students to publish more articles and get more involved with the community (with open-source contribution and suggestions). 

With most of the changes coming to completion - we will be sure to get your article published asap. 

Thank you again - and we are looking forward to more amazing articles.

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by hectorkambow** [comment]
@nellyat any progress or update on this PR. 

cc @WanjaMIKE

**Comment by WanjaMIKE** [comment]
Pushing test commit

**Pull request closed without merging**